WikiWatershed/model-my-watershed
# Compare View Revamp: Import Prototype

## Overview

Imports prototype from [`jdf/compare-prototype`](https://github.com/WikiWatershed/model-my-watershed/tree/jdf/compare-prototype), and decomposes into Marionette views, models, and templates.

This is rough, and meant to unblock other efforts. We can fix issues found herein later.

I couldn't get to the chart templates. Those are preserved in [`compareWindow2.html`](https://github.com/WikiWatershed/model-my-watershed/blob/ee7d3dad760b85bc3bc5cad3920c6336491b2161/src/mmw/js/src/compare/templates/compareWindow2.html#L34-L93) template.

I also did not remove any of the existing Compare stuff, even though it is effectively inaccessible. We can do so once the Compare View Revamp is complete.

Connects #2070 

### Demo

![image](https://user-images.githubusercontent.com/1430060/28683057-23d823c6-72cd-11e7-830c-afe7c98fb8f8.png)

## Testing Instructions

 * Check out this branch, and `bundle --debug --vendor`
 * Create / open a TR-55 project with more than once scenario.
 * Hit compare. You should see a view akin to the screenshot above.
